hack/tests/e2e-go.sh

```diff
@@ -7,15 +7,23 @@ set -o pipefail
 source ./hack/lib/test_lib.sh
 source ./hack/lib/image_lib.sh
 
+# install SDK binaries
```

Comment:
Why are any of these changes necessary?

Reply:
when we run the script locally if we do not `make install` first it will not get the changes
My idea/suggestion proposed in another pr was to keep the make install in the script instead of Travis.